app/src/main/java/org/android/go/sopt/App.kt

```diff
@@ -0,0 +1,16 @@
+package org.android.go.sopt
+
+import android.app.Application
+import org.android.go.sopt.util.PreferenceManager
+
+class App : Application(){
+
+    companion object{
+      lateinit var prefs : PreferenceManager
+    }
+
+    override fun onCreate() {
+        prefs = PreferenceManager(applicationContext)
+        super.onCreate()
+    }
+}
```

Comment:
```suggestion
class App : Application() {
    override fun onCreate() {
        prefs = PreferenceManager(applicationContext)
        super.onCreate()
    }
    
    companion object {
        lateinit var prefs : PreferenceManager
    }
}
```

companion object는 클래스 하단에 작성하는 것이 관례입니다.
[안드로이드 공식 문서](https://developer.android.com/kotlin/common-patterns#companion)와 [공식 문서에서 샘플로 제공하고 있는 sunflower 프로젝트의 코드](https://github.com/android/sunflower/blob/cdb3b2b5938ec0934ecaee800754c8e462c2d85c/app/src/main/java/com/google/samples/apps/sunflower/data/UnsplashRepository.kt#L35)를 참고해주세요!